docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - mongo
   dashboard:
     image: parseplatform/parse-dashboard
-    command: --dev --appId APPLICATION_ID --masterKey MASTER_KEY --serverURL "http://localhost:1337/parse" --appName affinity
+    command: --dev --appId APPLICATION_ID --masterKey MASTER_KEY --serverURL VUE_APP_PARSE_URL --appName affinity
```

Comment:
I don't think this works. this is not using a variable...

Reply:
fixed by @gnunicorn in https://github.com/TeamFranka/affinity/commit/fc54ca4b8bab645939860b80d8e85ade03aa539f